package.json

```diff
@@ -22,22 +22,21 @@
     "api-module-library/*"
   ],
   "devDependencies": {
-    "@auto-it/all-contributors": "^11.0.4",
-    "@auto-it/conventional-commits": "^11.0.4",
-    "@auto-it/first-time-contributor": "^11.0.4",
-    "@auto-it/slack": "^11.0.4",
-    "auto": "^11.0.4",
+    "@auto-it/all-contributors": "^11.1.1",
+    "@auto-it/conventional-commits": "^11.1.1",
+    "@auto-it/first-time-contributor": "^11.1.1",
+    "@auto-it/slack": "^11.1.1",
+    "auto": "^11.1.1",
     "lerna": "^7.4.2",
-    "nx": "^17.2.6"
+    "nx": "^16.10.0"
```

Comment:
Oh interesting, is nx 17 not compatible?

Reply:
yes it's not recommended with that lerna version, which is the highest that auto explicitly supports